index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.7/css/all.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>youTube Homepage</title>
+</head>
+<body>
+    <header class="flex-header">
+        <nav class="flex-header">
+            <div class="flex-header">
+               <div><a href=""><i class="fas fa-bars"></i></a></div>
+                <div> <a href=""><img src="image/logo.jpg" alt="youTube-logo" class="logo"></a></div>
+            </div>
+            <div class="flex-header">
+                <div><input type="search" value="search" class="search"></div>
+                <div><input type="button"  value="click" class="button"></div> 
+            </div>
+
+         <div class="flex-header">
+            <div> 
+                <div><a href="#"><img src="image/video-post.jpg" alt="right-nav" class="post"></a></div>
+            </div>
+            <div><div><a href="#"><img src="image/app.jpg" alt="right-nav"></a></div>
+            </div>
+            <div> 
+                <div> <a href="#"><img src="image/bell.jpg" alt="right-nav"></a></div>
+            </div>
+        </div>
+        </nav>
+    </header>
+    <main>
+        <div class="mediaPlayer"><iframe width="650" height="400" src="https://www.youtube.com/embed/V74l_zS1x8E" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+            <h3 class="p">Contributing to Open Source Part 1: The Essay Way</h3>
+               <div class="grid-cue">
+                   <div><p>57,800 Views * june 9, 2020</p></div>
+                   <div><i class="fas fa-thumbs-up"></i></div>
+                   <div><i class="fas fa-thumbs-down"></i></div>
+                   <div class="baby-flex"><i class="fas fa-share"></i><div>share</div></div>
+               </div>
+        </div>
+        <div class="listedView">
+            <div class="iconcue"><img src="image/firstfoto.jpg" alt=""></div> 
+            <div><img src="image/img5.jpg" alt="" class="iconcue"></div>
+            <div><img src="image/img4.jpg" alt="" class="iconcue"></div>
```

Comment:
Man you can do that, you should add your own CSS and HTML not getting captures from the original page